extensions/wikia/WikiaInYourLang/WikiaInYourLangController.class.php

```diff
@@ -65,8 +69,17 @@ public function getNativeWikiaInfo() {
 						$oNativeWiki->city_url,
 						$oNativeWiki->city_title,
 					];
-
-					$sMessagesAry = $this->prepareMessage( $sTargetLanguage, $aMessageParams );
+					$isMainPageLink = true;
+					if ( $wgWikiaEnvironment !== WIKIA_ENV_DEV ) {
+						$articleURL = $this->getArticleURL( $sArticleTitle, $oNativeWiki->city_id );
+						if ( $articleURL ) {
+							$aMessageParams[1] = $articleURL;
+							$this->response->setVal( 'linkAddress', $articleURL );
+							$isMainPageLink = false;
+						}
+					}
+
+					$sMessagesAry = $this->prepareMessage( $sTargetLanguage, $aMessageParams, $isMainPageLink );
```

Comment:
This code is using Hungarian notation (though we don't have any convention of doing (or not) doing so), but if it is going to use Hungarian notation, this variable is an array not a string, so should be something like `$aMessages`.